examples/Azure_IoT_Central_ESP32_AzureIoTKit/Azure_IoT_Central_ESP32_AzureIoTKit.ino

```diff
@@ -141,11 +141,9 @@ static int mqtt_client_init_function(mqtt_client_config_t* mqtt_client_config, m
   mqtt_config.disable_auto_reconnect = false;
   mqtt_config.event_handle = esp_mqtt_event_handler;
   mqtt_config.user_context = NULL;
-  mqtt_config.buffer_size = 1024;
```

Comment:
interesting....this isn't needed anymore?

Reply:
1024 is already the default internal size of the ESP MQTT buffer.
https://github.com/espressif/esp-mqtt/blob/master/lib/include/mqtt_config.h#L27
https://github.com/espressif/esp-mqtt/blob/master/mqtt_client.c#L786